src/PowerShellEditorServices.Hosting/EditorServicesLoader.cs

```diff
@@ -98,7 +98,7 @@ public sealed class EditorServicesLoader : IDisposable
 
             AssemblyLoadContext.Default.Resolving += (AssemblyLoadContext _, AssemblyName asmName) =>
             {
-#if DEBUG
+#if ASSEMBLY_LOAD_STACKTRACE
```

Comment:
Yesssssssssss

Reply:
Was _really_ bugging me, but didn't want to throw this away either.